build/deb_rpm/v2-jf/build-scripts/pack.sh

```diff
@@ -64,15 +63,15 @@ createDEBPackage(){
 	local flavour="deb"
 
 	# cleanup old files and containers
-	rm -f ${JFROG_CLI_PKG}/${JFROG_CLI_PREFIX}*${VERSION_FORMATTED}*.${flavour}
+	rm -f "${JFROG_CLI_PKG}"/${JFROG_CLI_PREFIX}*"${VERSION_FORMATTED}"*.${flavour}
```

Comment:
Can't we put the entire string under double quotes?
```suggestion
	rm -f "${JFROG_CLI_PKG}/${JFROG_CLI_PREFIX}*${VERSION_FORMATTED}*.${flavour}"
```